### PR TITLE
Move to use published fabric-protos node modules

### PR DESCRIPTION
**Pull request by mbwhite**
Move the codebase to directly using the published protobuf libraries. The fabric-protos
repo, as well as holding the 'main' copy of the protos, also publishes packages for each language
These can then be pulled directly here; everything is under version control.

Removes the need for the copies of the protobufs held locally in this repo.

Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

### CODE REVIEW

File: libraries/fabric-shim/test/unit/utils/statebased.js [hunk 1]
```diff
@@ -72,45 +72,48 @@ describe('KeyEndorsementPolicy', () => {
...
-            const ep = new KeyEndorsementPolicy();
-            ep._setMspIdsFromSPE(spe);
-            expect(ep.orgs).haveOwnProperty('Org1MSP').eqls(0);
-            expect(ep.orgs).haveOwnProperty('Org2MSP').eqls(0);
-            expect(ep.orgs).haveOwnProperty('Org3MSP').eqls(0);
-            expect(ep.orgs).haveOwnProperty('Org4MSP').eqls(3);
-            expect(ep.orgs).haveOwnProperty('Org5MSP').eqls(3);
-        });
-    });
+    // describe('#_getPolicyFromMspId', () => {
```
Comment (jt-nti) [inline libraries/fabric-shim/test/unit/utils/statebased.js:75]: Commented out code

File: libraries/fabric-shim/test/unit/stub.js [hunk 2]
```diff
@@ -68,19 +69,19 @@ describe('Stub', () => {
...
+                            getEpoch: () => {
+                                return {high: 10, low: 1};
                             }
                         }
                     }
                 }
             };
 
-            expect(computeProposalBinding(decodedSP)).to.deep.equal('ff7e9beabf035d45cb5922278f423ba92f1e85d43d54c2304038f2f2b131625b');
+            // expect(computeProposalBinding(decodedSP)).to.deep.equal('ff7e9beabf035d45cb5922278f423ba92f1e85d43d54c2304038f2f2b131625b');
```
Comment (jt-nti) [inline libraries/fabric-shim/test/unit/stub.js:83]: Commented out code

**Review by jt-nti — CHANGES_REQUESTED** [review]
Couple of minor comments but looks good to me- fantastic to be able to get rid of the .proto copies!

File: libraries/fabric-shim/lib/handler.js [hunk 3]
```diff
@@ -391,162 +375,206 @@ class ChaincodeMessageHandler {
...
-                    args: argsAsBuffers
-                }
-            }).finish(),
+        const msgPb = new peer.ChaincodeSpec();
+        const chaincodeIdPb = new peer.ChaincodeID();
+        chaincodeIdPb.setName(chaincodeName);
+        msgPb.setChaincodeId(chaincodeIdPb);
+
+        const chaincodeInputPb = new peer.ChaincodeInput();
+        chaincodeInputPb.setArgsList(args.map((value) => Buffer.from(value)));
```
[the remark this answers is not in the document]
Comment (mbwhite) [reply libraries/fabric-shim/lib/handler.js:544]: 'utf8' is the default for Buffer.from()

In reality, though, that is my 3rd version of the function; so the 'utf8' dropped out along the way...

File: libraries/fabric-shim/lib/server.js [hunk 4]
```diff
@@ -6,30 +6,30 @@
...
-        defaults: true,
-        oneofs: true,
-        includeDirs: [
-            path.resolve(__dirname, '..', 'google-protos'),
-            path.resolve(__dirname, '..', 'protos')
-        ]
-    }
-);
-const protoDescriptor = grpc.loadPackageDefinition(packageDefinition);
+// const PROTO_PATH = path.resolve(__dirname, '..', 'protos', 'peer', 'chaincode_shim.proto');
```
[the remark this answers is not in the document]
Comment (mbwhite) [reply libraries/fabric-shim/lib/server.js:17]: Honestly surprised there wasn't more...

**Review by jt-nti — APPROVED** [review]

**Pull request merged**